src/epcpm/mainwindow.ui

```diff
@@ -151,8 +168,8 @@
     <rect>
      <x>0</x>
      <y>0</y>
-     <width>1200</width>
-     <height>20</height>
+     <width>1516</width>
+     <height>21</height>
```

Comment:
Unless there is a reason to change the size, I'd leave these alone.

Reply:
These changes were added automatically by the Qt designer tool. Reverted.